ftl/common/ftl_util.py

```diff
@@ -27,3 +31,18 @@ def __enter__(self):
     def __exit__(self, unused_type, unused_value, unused_traceback):
         end = time.time()
         logging.info('%s took %d seconds', self.descriptor, end - self.start)
+
+
+def folder_to_layer_sha(pkg_dir, pkg_mngr_name):
```

Comment:
Maybe "zip_directory" would be more clear?

Reply:
"zip_directory_to_layer_sha"?  This takes a path to a folder that is uncompressed/untarred

Reply:
done